src/mappings/rewardsDistributor.ts

```diff
@@ -47,32 +47,32 @@ export async function handleRewardsDistributed(
     api
   );
 
-  await Promise.all(
-    delegators.map(async (delegator) => {
-      const rewards = await rewardsDistributor.userRewards(
-        indexer,
-        delegator.delegatorId
-      );
-      const id = buildRewardId(indexer, delegator.delegatorId);
-
-      let reward = await UnclaimedReward.get(id);
-
-      if (!reward) {
-        reward = UnclaimedReward.create({
-          id,
-          delegatorAddress: delegator.delegatorId,
-          indexerAddress: indexer,
-          amount: rewards.toBigInt(),
-          createdBlock: event.blockNumber,
-        });
-      } else {
-        reward.amount = rewards.toBigInt();
-        reward.lastEvent = `handleRewardsDistributed:${event.blockNumber}`;
-      }
-
-      await reward.save();
-    })
-  );
+  for (const delegator of delegators.sort((a, b) =>
+    a.delegatorId.localeCompare(b.delegatorId)
```

Comment:
Any reason we need to sort delegators?

Reply:
The order of delegators we get from db might vary among different indexers.